flowzone.yml

```diff
@@ -3367,6 +3367,7 @@ jobs:
             {
               "metadata": "read",
               "contents": "read",
+              "issues": "write",
```

Comment:
`"pull_requests": "write"` should be enough, based on testing I did elsewhere.

Reply:
Is this not enough? https://github.com/balena-io/balena-cloud/blob/e0a5401062a8e8799227e9d8f231fbf1d3078366/.github/workflows/flowzone.yml#L21C1-L21C49

Reply:
Hmm, it is using the [listComments](https://github.com/peter-evans/find-comment/blob/main/src/find.ts) REST endpoint which may need more than actually posting a comment. @ab77 why don't we try just giving it "issues": "read"?

Reply:
I just took the settings from [here](https://github.com/peter-evans/create-or-update-comment)

<img width="937" alt="image" src="https://github.com/user-attachments/assets/39c6b6e7-270c-4391-a6f6-0873d705cc02">

Reply:
I think try it with just `issues:read` and that is hopefully enough.

Reply:
Let's try this instead: https://github.com/product-os/flowzone/pull/1252